pandas/_libs/groupby.pyx

```diff
@@ -511,9 +511,9 @@ def group_shift_indexer(
 @cython.wraparound(False)
 @cython.boundscheck(False)
 def group_fillna_indexer(
-    ndarray[intp_t] out,
-    ndarray[intp_t] labels,
-    ndarray[uint8_t] mask,
+    intp_t[::1] out,
```

Comment:
Since intp is a NumPy thing it would be nice to remove that here too. `ndarray[intp_t]` should be equivalent to `Py_ssize_t[]`

Reply:
Yup using `Py_ssize_t` here worked. Changed to that